### PR TITLE
Add missing reqs for TTS on android

### LINKED ISSUE #1098: Platform specific requirements (text-to-speech)
I couldn't find it documented anywhere (may have missed it), but there are platform specific requirements for this to work.
For example, for android, the following intent needs adding to the AndroidManifest.xml file
```
		<intent>
			<action android:name="android.intent.action.TTS_SERVICE" />
		</intent>

```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 80172681-b0f2-26ea-9062-b23c15699ed4
* Version Independent ID: 80172681-b0f2-26ea-9062-b23c15699ed4
* Content: [Text-to-Speech - .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/device-media/text-to-speech?view=net-maui-7.0)
* Content Source: [docs/platform-integration/device-media/text-to-speech.md](https://github.com/dotnet/docs-maui/blob/main/docs/platform-integration/device-media/text-to-speech.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

---
[Associated WorkItem - 45532](https://dev.azure.com/msft-skilling/Content/_workitems/edit/45532)

### PR DESCRIPTION
**Pull request by adegeo**
Also, I removed the stupid markdown lint rule that interferes with tabs.

Fixes #1098

### CODE REVIEW

File: docs/platform-integration/device-media/text-to-speech.md [hunk 1]
```diff
@@ -11,6 +11,34 @@ This article describes how you can use the .NET Multi-platform App UI (.NET MAUI
 
 The default implementation of the `ITextToSpeech` interface is available through the `TextToSpeech.Default` property. Both the `ITextToSpeech` interface and `TextToSpeech` class are contained in the `Microsoft.Maui.Media` namespace.
 
+## Get started
+
+To access text-to-speech functionality, the following platform-specific setup is required.
+
+# [Android](#tab/android)
+
+If your project's Target Android version is set to **Android 11 (R API 30)** or higher, you must update your _Android Manifest_ with queries that use Android's [package visibility requirements](https://developer.android.com/preview/privacy/package-visibility).
```
Comment (davidbritch) [inline docs/platform-integration/device-media/text-to-speech.md]: This URL redirects to https://developer.android.com/about/versions/11/privacy/package-visibility. But that page is also relatively useless and invites you to navigate to https://developer.android.com/training/package-visibility. So maybe that's the URL to link to.

Comment (davidbritch) [reply docs/platform-integration/device-media/text-to-speech.md]: Also think it's better to say something like "....you must update your _Android Manifest_ with an intent filter for the text-to-speech (TTS) engine...".

**Review by davidbritch — APPROVED** [review]
Approving so you're unblocked.

File: docs/platform-integration/device-media/text-to-speech.md (hunk 1 above)
Comment (adegeo) [reply docs/platform-integration/device-media/text-to-speech.md]:
```suggestion
If your project's Target Android version is set to **Android 11 (R API 30)** or higher, you must update your _Android Manifest_ with an intent filter for the text-to-speech (TTS) engine. For more information about intents, see Android's documentation on [Intents and Intent Filters](https://developer.android.com/guide/components/intents-filters).
```

**Pull request merged**